userguide/platform/userguide_deployment.adoc

```diff
@@ -79,13 +79,17 @@ org.killbill.billing.osgi.dao.user=killbill
 org.killbill.billing.osgi.dao.password=killbill
 ----
 
-Download the current stable version of the Kill Bill war from https://search.maven.org/search?q=a:killbill-profiles-killbill[Maven Central] and place it under *CATALINA_HOME/webapps/ROOT.war* (delete the directory *CATALINA_HOME/webapps/ROOT* if it exists).
+Download the current stable version of the Kill Bill war from https://search.maven.org/search?q=a:killbill-profiles-killbill[Maven Central] (make sure to download the `killbill-profiles-killbill-X.Y.Z.war` file and not the `jetty-console.war`, `jar-with-dependencies.war` or `jar-with-dependencies-sources.war` one) and place it under *CATALINA_HOME/webapps/ROOT.war* (delete the directory *CATALINA_HOME/webapps/ROOT* if it exists).
```

Comment:
![Screen Shot 2019-11-07 at 11 04 10 AM](https://user-images.githubusercontent.com/246910/68419179-96f5f600-014e-11ea-9753-b2afd3e99220.png)


When we look at the link -- see above -- we see a list of option. I suppose from that list, this is the one that shows as `war`. Might be worth making this explicit.